src/platform/tests/TestCHIPoBLEStackMgr.cpp

```diff
@@ -36,6 +37,8 @@ void EventHandler(const chip::DeviceLayer::ChipDeviceEvent * event, intptr_t arg
 
 int TestCHIPoBLEStackManager()
 {
+    chip::Platform::MemoryInit();
```

Comment:
I wish this does not need to be added to all tests manually.

Reply:
I believe it depends if memory is used or not. I blindly copied stuff that was extra in 'Driver' into the test code.